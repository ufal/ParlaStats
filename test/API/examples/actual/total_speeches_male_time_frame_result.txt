[{'person.sex': 'M', 'total_speeches': 285}]

Running time 4.737060546875 ms

+------------+----------------+
| person.sex | total_speeches |
+------------+----------------+
|     M      |      285       |
+------------+----------------+
