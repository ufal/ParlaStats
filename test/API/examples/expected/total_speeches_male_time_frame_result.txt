[{'person.sex': 'M', 'total_speeches': 285}]

Running time 5.8994140625 ms

+------------+----------------+
| person.sex | total_speeches |
+------------+----------------+
|     M      |      285       |
+------------+----------------+
